Automattic/jetpack
# Payment Blocks: Add fees and management link to product management controls

Fixes #23569

#### Changes proposed in this Pull Request:

* Added the "Manage your subscriptions" link inside the subscription panel.
* Added a link to "Read more about fees" inside the subscription panel.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Create or edit a post.
* Add a premium content block.
* In the sidebar, you should see a link to "Manage your subscriptions" and "Read more about fees"

#### Test evidence:
https://user-images.githubusercontent.com/1989914/159687565-6ebc329a-5dca-48d6-947a-b4463e37b375.mp4
